packages/sync-repo-settings/src/sync-repo-settings.ts

```diff
@@ -26,7 +26,7 @@ import Ajv from 'ajv';
 import yaml from 'js-yaml';
 import {PullsListFilesResponseData} from '@octokit/types';
 
-export const configFileName = 'sync-repo-settings.yaml';
+export const configFileName = 'sync-repo-settings.yml';
```

Comment:
There seem to be some changes unrelated?

Reply:
oof, thanks for the heads up. Need to resync to main branch.